### PR TITLE
some changes in the bugreporting page to reflect the new redmine forms

### PR DESCRIPTION
**Pull request by gioman**

### CODE REVIEW

File: source/site/getinvolved/development/bugreporting.rst [hunk 1]
```diff
@@ -72,15 +72,15 @@ Important information needed when opening a ticket:
...
+            or if it is a regression (a regression is an issue about a functionality that
+            used to work as expected and then stopped to work in more recent QGIS release)
 * **Category** - Choose the aspect of the application that is most closely
   associated with the problem
-* **Milestone** - If this issue affects a particular Milestone in the project
-  choose it from the drop-down list
-* **Version** - Version of QGIS this issue affects
-* **Platform** - Choose the platform you are using
+* **Affected QGIS Version** - Version of QGIS this issue affects
+* **Operating System** - Choose the platform you are using
```
Comment (yjacolin) [inline source/site/getinvolved/development/bugreporting.rst]: `Choose the operating system you are using?` ?

**Review by yjacolin — CHANGES_REQUESTED** [review]

**Review by yjacolin — APPROVED** [review]

File: source/site/getinvolved/development/bugreporting.rst (hunk 1 above)
Comment (gioman) [reply source/site/getinvolved/development/bugreporting.rst]: good catch, I changed the wording.

**Comment by yjacolin** [comment]
@gioman Feel free to merge after the checks are green or ping me if you have no commit right :) Thanks!

**Comment by gioman** [comment]
@yjacolin I don't commit rights, can you merge this? thanks!

File: source/site/getinvolved/development/bugreporting.rst [hunk 2]
```diff
@@ -72,15 +72,16 @@ Important information needed when opening a ticket:
...
   * *Low*: a problem which doesn't affect QGIS usefulness, 
   * *Normal*: the default value, applicable to most bugs and nearly all feature requests, 
-  * *High*: a bug which has a major effect on the usability of a package, 
-  * or *Severe/Regression*: a bug that makes QGIS totally unusable, causes serious 
-    data loss or a regression from a previous QGIS version
+  * *High*: a bug which has a major effect on the usability of a package or
+            a bug that makes QGIS totally unusable. In the form you will find also
+            yes/no fields to allow you detail of the issue is causing a crash or data corruption
+            or if it is a regression (a regression is an issue about a functionality that
+            used to work as expected and then stopped to work in more recent QGIS release)
```
Comment (DelazJ) [inline source/site/getinvolved/development/bugreporting.rst]: these three lines should be aligned on \*High\* (see line 84)

Comment (gioman) [reply source/site/getinvolved/development/bugreporting.rst]: should be ok now

**Comment by DelazJ** [comment]
Thanks

**Pull request merged**